test-requirements.txt

```diff
@@ -1,4 +1,3 @@
-pytest==6.2.4
 PyYAML==6.0
 docker==5.0.0
 six==1.16.0
```

Comment:
Let's just remove this file and move the other things to `dev-requirements.txt`. It's not needed & confusing

Reply:
done